site/src/components/HomepageFeatures/index.tsx

```diff
@@ -0,0 +1,68 @@
+import CodeBlock from '@theme/CodeBlock';
+import Heading from '@theme/Heading';
+import clsx from 'clsx';
+import { ComponentProps, ComponentType, JSX } from 'react';
+import styles from './styles.module.css';
+
+import ImageIcon from '@site/static/img/image.svg';
+import SoundIcon from '@site/static/img/sound-on.svg';
+import TextIcon from '@site/static/img/text.svg';
+
+type FeatureItem = {
+  title: string;
+  Icon: ComponentType<ComponentProps<'svg'>>;
+  code: string;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: 'Text Generation API',
+    Icon: TextIcon,
+    code: `ov_pipe = ov_genai.LLMPipeline("TinyLlama")
+print(ov_pipe.generate("The Sun is yellow because"))`,
+  },
+  {
+    title: 'Image Generation API',
+    Icon: ImageIcon,
+    code: `ov_pipe = ov_genai.Text2ImagePipeline("Flux")
+image = ov_pipe.generate("Create beautiful Sun")`,
+  },
+  {
+    title: 'Speech to Text API',
+    Icon: SoundIcon,
+    code: `ov_pipe = ov_genai.WhisperPipeline("whisper-base")
+print(ov_pipe.generate(read_wav("sample.wav)))`,
+  },
+];
+
+function Feature({ title, Icon, code }: FeatureItem) {
+  return (
+    <div>
+      <div className={clsx('text--center', styles.featureHeader)}>
```

Comment:
I found 2 places where `clsx` used. Both of them are not conditional. Do we really need `clsx` package?

Reply:
Actually it is a dependency of docusaurus package, I will delete it from our package.json